Merge Skinned Mesh Enablement Animation (#1173)
anatawa12/AvatarOptimizer: CHANGELOG.md

    - We may relax some restriction in the future.
  - Because we have to check for each condition if we use AnyState but we can check for only one (in best case) with entry/exit, this generally reduces cost for checking an parameter in a state.
  - Combined with Entry / Exit to 1D BlendTree optimization, which is implemented in previous release, your AnyState layer may be optimized to 1D BlendTree.
- `Copy Enablement Animation` to Merge Skinned Mesh `#1173`

>   - This feature copies activeness / enablement animation from source renderers to the merged renderer.

Reviewer: ```suggestion
  - This feature copies activeness / enablement animation from merge target renderers to the merged renderer.
```

  - This feature is not enabled by default. You have to enable it in the inspector.

>   - This feature supports copying activeness animation of `activeSelf` of the GameObject or ancestors of the GameObject,

Reviewer: ```suggestion
  - This feature supports copying activeness animation of `activeSelf` of the GameObjects or ancestors of the GameObjects.
```

    However, this feature does not work if multiple GameObjects (or both GameObject and Renderer itself) are animated.

>   - In addition, this feature will be animate the `enabled` of the merged renderer, so you must not animate the `enabled` of the merged renderer.

Reviewer: ```suggestion
  - In addition, this feature will be animate the `enabled` of the merged renderer, so you must not animate the `enabled` of the merged renderer.
```

Author: no diff?

Reviewer: そうっぽい(よく見ずにprereleaseからコピペした)

> - `Copy Enablement Animation` to Merge Skinned Mesh `#1173`
>   - This feature copies activeness / enablement animation from source renderers to the merged renderer.
>   - This feature is not enabled by default. You have to enable it in the inspector.
>   - This feature supports copying activeness animation of `isActive` of the GameObject or ancestors of the GameObject,
>     However, this feature does not work if multiple GameObjects (or both GameObject and Renderer itself) are animated.
>   - In addition, this feature will be animate the `enabled` of the merged Renderer, so you must not animate the Enabled of the merged Renderer.
>     - If animations are unsupported, AAO will show a error message and abort the build.

Author: ```suggestion
- `Copy Enablement Animation` to Merge Skinned Mesh `#1173`
  - This feature copies activeness / enablement animation from source renderers to the merged renderer.
  - This feature is not enabled by default. You have to enable it in the inspector.
  - This feature supports copying activeness animation of `activeSelf` of the GameObject or ancestors of the GameObject,
    However, this feature does not work if multiple GameObjects (or both GameObject and Renderer itself) are animated.
  - In addition, this feature will be animate the `enabled` of the merged renderer, so you must not animate the `enabled` of the merged renderer.
    - If animations are unsupported, AAO will show a error message and abort the build.
```

>     - If animations are unsupported, AAO will show a error message and abort the build.

Reviewer: ```suggestion
    - If animations are unsupported, AAO will show an error message and abort the build.
```
